frontend/app/src/connection/WebsocketConnection.test.tsx

```diff
@@ -34,7 +34,7 @@ import {
   Args,
 } from "@streamlit/app/src/connection/WebsocketConnection"
 
-const MOCK_ALLOWED_ORIGINS_RESPONSE = {
+const MOCK_ALLOWED_HOST_CONFIG_RESPONSE = {
```

Comment:
nit: should this be `MOCK_HOST_CONFIG_RESPONSE`?

We may also want to keep this as `MOCK_ALLOWED_ORIGINS_RESPONSE`  and have a separate `MOCK_HOST_CONFIG_RESPONSE` since it only happens to coincide with the host config response type right now but won't in the future.

Reply:
Yep, I split this up now into `MOCK_ALLOWED_ORIGINS_CONFIG` and `MOCK_HOST_CONFIG_RESPONSE` to make this a bit more separated.